_includes/_pagination.html

```diff
@@ -13,8 +13,19 @@
   <div class="load-more">
     <button class="btn {% if include.btn-type == 'white-outline' %}btn-white btn-outline{% else %}btn-blue{% endif %}" data-role="load-more" data-paginate="{{ include.collection }}">Load more{{ include.label | prepend: ' ' }}</button>
   </div>
-  {% endif %}
 
+  {% elsif include.n_of_total_pagination == "true" %}
+  <div class="text-center input-pagination">
+    <button class="btn btn-blue" id="pagination-previous-button"><<</button>
+    <input class="form-control form-control-custom-small" type="number"
+    max="{{paginator.total_pages}}" id="pagination-current-page" placeholder="{{paginator.page}}">
+    <span>of</span>
+    <span id="total_pages">{{paginator.total_pages}}</span>
+    <button class="btn" id="pagination-submit-button">Go!</button>
+    <button class="btn btn-blue" id="pagination-next-button">>></button>
```

Comment:
- I converted the `>>` to `>`. I usually see `>` for go to the next page and `>` is used for the last page.
- I converted the button blue background to be just blue text. This will go with the submit button changes.
- I converted `>` to `&gt;` to separate it from the HTML tags.

```suggestion
    <button class="btn btn-link text-blue" id="pagination-next-button">&gt;</button>
```